dev-infrastructure/docs/development-setup.md

```diff
@@ -259,6 +259,13 @@ Option 2: You can follow the below manual steps from the root of the CS repo on
 # Setup the development.yml
 cp ./configs/development.yml .
 
+# Setup the azure-runtime-config.json
+cp ./configs/azure/example-config.json ./azure-runtime-config.json
```

Comment:
not clear here but the azure-runtime-config, the cert and the client id all needs to be set in the `development.yml` file. I would add a note to review the required properties there, it should be documented in the development.yml file itself. The section for getting the values for those required properties can be moved alongside that section. 

For example, FPA_CLIENT_ID env var is set below but it's not really used anywhere and it's not clear what/where it should be used for.